ibc-testkit/tests/core/ics02_client/update_client.rs

```diff
@@ -801,6 +803,7 @@ fn test_misbehaviour_client_ok(fixture: Fixture, #[case] msg_envelope: MsgEnvelo
     ensure_misbehaviour(&ctx, &client_id, &mock_client_type());
 }
 
+#[allow(deprecated)]
 #[rstest]
 fn test_submit_misbehaviour_nonexisting_client(fixture: Fixture) {
```

Comment:
For the following tests:

- test_submit_misbehaviour_nonexisting_client
- test_misbehaviour_synthetic_tendermint_equivocation
- test_misbehaviour_synthetic_tendermint_bft_time

I suggest we switch to the `MsgUpdateClient` for constructing misbehavior messages as that's our main route from now on.

Reply:
[f2d2056](https://github.com/cosmos/ibc-rs/pull/1079/commits/f2d20567af1324b1537aea6d704882597fa85e5b)